apps/modernization-ui/src/design-system/select/single/SingleSelect.spec.tsx

```diff
@@ -59,15 +66,21 @@ describe('when one of the options is clicked', () => {
                     { name: 'name-four', value: 'value-four', label: 'label-four' }
                 ]}
                 name="test-name"
+                value={value}
+                onChange={onChange}
             />
         );
+        let selectedValue = null;
+        const handleChange = (value?: Selectable) => {
+            selectedValue = value || null;
+        };
 
-        const select = getByRole('combobox', { name: 'Test Label' });
+        const { getByRole } = render(<SingleSelectTestWrapper value={selectedValue} onChange={handleChange} />);
 
-        userEvent.selectOptions(select, 'name-four');
+        const select = getByRole('combobox', { name: 'Test Label' });
 
-        const checked = getByRole('option', { selected: true });
```

Comment:
issue: The `getByRole('option', { selected: true });` is testing that the `aria-checked` property is applied to the selected option.  Was this test change because it was failing after changes were made to `SelectInput`?